<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>376</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>376</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>547</width>
    <height>376</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>SF Pro Rounded</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(48, 48, 48);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>151</width>
      <height>7</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>7</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>7</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
selection-color: #000000;
gridline-color: #303030;
background-color: #303030;
selection-background-color: #ffffff;
border-style: solid;
border-radius: 10px;
border-color: #393939;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PauseButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PrevButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NextButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listTrack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>200</width>
      <height>301</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="openFolderButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Open Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Pro Rounded</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Music Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #393939;
selection-background-color: #999999;
selection-color: #ffffff;
color: #ffffff;
border-width: 2px;
border-style: solid;
border-radius: 10px;
border-color: #393939;
alternate-background-color: #303030;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>392</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
